app/src/main/res/layout/fragment_user.xml

```diff
@@ -1,24 +1,200 @@
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/frag_user_recyclerView"
+            android:layout_width="0dp"
+            android:layout_height="140dp"
+            android:orientation="vertical"
+            android:layout_marginTop="@dimen/big_size"
+            app:layout_constraintTop_toBottomOf="@+id/frag_user_tv_blanc1"
+            app:layout_constraintStart_toStartOf="@+id/layout_user"
+            app:layout_constraintEnd_toEndOf="@+id/layout_user"
+            />
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/frag_user_tv_blanc4"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/big_size"
+            android:layout_marginTop="@dimen/big_size"
+            android:layout_marginEnd="@dimen/medium_size"
+            android:layout_marginStart="@dimen/medium_size"
+            android:background="@color/buscar_edit_text"
+            app:layout_constraintTop_toBottomOf="@id/frag_user_recyclerView"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"/>
+
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/frag_user_tv_list_follow"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Lista de seguimiento"
+            android:textColor="@color/black"
+            android:layout_marginTop="@dimen/big_size"
+            android:textSize="@dimen/big_size_text"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="@+id/frag_user_recyclerView"
+            app:layout_constraintTop_toBottomOf="@id/frag_user_tv_blanc4"/>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/frag_user_tv_list_follow_description"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/medium_size"
+            android:text="Crea una lista de seguimiento para no perderte ninguna pelicula o serie de tv"
+            android:textSize="15dp"
+            app:layout_constraintTop_toBottomOf="@id/frag_user_tv_list_follow"
+            app:layout_constraintEnd_toEndOf="@id/frag_user_recyclerView"
+            app:layout_constraintStart_toStartOf="@id/frag_user_recyclerView"
+            />
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/frag_user_tv_list_follow_bottom"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:background="@drawable/style_border"
+            android:backgroundTint="@color/colorApp"
+            android:layout_marginTop="@dimen/medium_size"
+            android:textAppearance="@style/TextAppearance.AppCompat.Large"
+            android:textSize="18sp"
```

Comment:
in this case you're using the right dimension spec (sp) but please move it to the dimes file